<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>iconb.png</normaloff>iconb.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
}
QLineEdit{
border: 4px solid #ffee01;
border-radius: 7px;
}
QPushButton{
background: white;
border: 4px solid #ffee01;
border-radius: 7px;
}
QPushButton:pressed{
color: #ffee01;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="linUser">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linPass">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chckRem">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblRem">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remember Me</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblFooter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>801</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background: #ffee01;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 2px solid black;
background:  #ffee01;
border-radius: 0px;
}
QWidget:hover{
background: red;
color: white;
}
</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinimize">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>0</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 2px solid black;
background:  #ffee01;
border-radius: 0px;
}
QWidget:hover{
color: white;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeader">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>801</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background: #ffee01;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMaximize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>0</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 2px solid black;
background:  #ffee01;
border-radius: 0px;
}
QWidget:hover{
color: white;
}</string>
    </property>
    <property name="text">
     <string>□</string>
    </property>
   </widget>
   <zorder>lblHeader</zorder>
   <zorder>linUser</zorder>
   <zorder>linPass</zorder>
   <zorder>btnLogin</zorder>
   <zorder>chckRem</zorder>
   <zorder>lblRem</zorder>
   <zorder>lblLogo</zorder>
   <zorder>lblFooter</zorder>
   <zorder>btnExit</zorder>
   <zorder>btnMinimize</zorder>
   <zorder>btnMaximize</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
